packages/vite/src/node/server/index.ts

```diff
@@ -114,6 +115,13 @@ export interface ServerOptions extends CommonServerOptions {
    * in a future minor version without following semver
    */
   force?: boolean
+  /**
+   * Disable key bindings for the server by setting this to `false`. This can be
+   * useful if you need the `process.stdin` stream for another purpose.
+   *
+   * @default true
```

Comment:
This should default to false for programmatic usages and only default to true in CLI

Reply:
@antfu was this a docs-only change? Tbh I couldn't actually figure out where `bindShortcuts` was defaulted to `true` in the code. @aleclarson would you please point me in the right direction?